api/discovery/discovery.proto

```diff
@@ -62,13 +62,16 @@ message StartDiscoveryRequest { optional string resource_group = 1; }
 message StartDiscoveryResponse { bool successful = 1; }
 
 message QueryRequest {
-  optional string filtered_type = 1;
-  optional string filtered_cloud_service_id = 2;
+  int32 page_size = 1;
```

Comment:
This applies to all the `ListXX` methods

Reply:
Should I change the number convention (starting at 10) in every .proto file?

Reply:
Yes please

Reply:
Roger that

Reply:
Wait.. you misunderstood me. Please keep existing files how they are and revert the page_size back to index 10

Reply:
I am confused.
page_size = 10 in discovery.proto

Reply:
You changed the starting index of every RPC in this request to 10. This is not what I meant. I originally meant that we should keep the query options (page_size, etc.) beginning at 10, like they were before. All other properties start at 1, keeping the changes to the numbers as minimal as possible.

Reply:
To avoid further confusion I took the liberty to cleanup the proto index to the system we had previously. We probably should have a closer look at this and add it to the list of conventions and possibly discuss if we are happy with that convention.

Reply:
Thanks, we did the reordering in parallel -> Race Condition :D